miden-tx/src/tests/mod.rs

```diff
@@ -973,3 +872,13 @@ fn test_tx_script() {
         executed_transaction,
     );
 }
+
+fn create_fungible_asset() -> Asset {
+    Asset::Fungible(
+        FungibleAsset::new(
+            ACCOUNT_ID_FUNGIBLE_FAUCET_ON_CHAIN.try_into().expect("id is valid"),
+            FUNGIBLE_ASSET_AMOUNT / 2,
+        )
+        .expect("asset is valid"),
+    )
+}
```

Comment:
Should this be implemented as `FungibleAsset::mock()` or maybe `FungibleAsset::mock(amount)`.